lib/repeatable/parser.rb

```diff
@@ -38,6 +38,19 @@ def expression_for(key, value)
           included: build_expression(value[:included]),
           excluded: build_expression(value[:excluded])
         )
+      when Repeatable::Expression::EndsAt
+        value = symbolize_keys(value)
+        klass.new(
+          ends_at: value[:ends_at],
+          expression: build_expression(value[:expression])
+        )
+      when Repeatable::Expression::EndsAfter
+        value = symbolize_keys(value)
```

Comment:
I copied this from examples above, but on some Rubies, it could introduce a security vulnerabilities. If users pass external input into `Schedule.new`, external input will be converted into symbols. Then, symbols are never garbage collected (until a certain CRuby version?). If an attacker knows this, they can provide unique strings, which become symbols, until the system runs out of memory 😖